src/main/java/com/direwolf20/buildinggadgets/common/building/modes/TargetedAxisChasingMode.java

```diff
@@ -0,0 +1,45 @@
+package com.direwolf20.buildinggadgets.common.building.modes;
+
+import com.direwolf20.buildinggadgets.common.BuildingGadgets;
+import com.direwolf20.buildinggadgets.common.building.IPlacementSequence;
+import com.direwolf20.buildinggadgets.common.building.IValidatorFactory;
+import com.direwolf20.buildinggadgets.common.building.placement.Column;
+import com.direwolf20.buildinggadgets.common.building.placement.ExclusiveAxisChasing;
+import com.direwolf20.buildinggadgets.common.tools.IAtopSupport;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * <p>Formal "Build to Me"</p>
+ */
+public class TargetedAxisChasingMode extends AbstractMode implements IAtopSupport {
+
+    private static final ResourceLocation NAME = new ResourceLocation(BuildingGadgets.MODID, "axis_chasing");
+
+    public TargetedAxisChasingMode(IValidatorFactory validatorFactory) {
+        super(validatorFactory);
+    }
+
+    @Override
+    public IPlacementSequence computeCoordinates(EntityPlayer player, BlockPos hit, EnumFacing sideHit, ItemStack tool) {
+        BlockPos entityPos = new BlockPos(Math.floor(player.posX), Math.floor(player.posY), Math.floor(player.posZ));
+        //TODO add config "inclusivePlacementForBuildToMe" as a config option
```

Comment:
When

Reply:
next PR in 1.12 -- don't want add any extra featurea into this one.